### ISSUE TITLE
Member Profile > Administration > Access Overview Page Error (only when Security Lock on Primary Role of member?)

### ISSUE THREAD

**Issue by mcfarlandonline** [ExpressionEngine/ExpressionEngine#1459]
EE Version 6.1.4
PHP Version 7.3.30

When logged in as a Super Admin, I'm getting an error when trying to view a member's "Access Overview" page. 

Seems to only be the case if the member's Primary Role is assigned to a group that has the Security Lock turned on, (only Super Admins can add or remove members from this role). 

Screenshot of error is attached below, as well: 

> Error Caught
> 
> Call to a member function getName() on null
> 
> ee/ExpressionEngine/Controller/Members/Profile/Access.php:161
> 
> Stack Trace: Please include when reporting this error
> 
> #0 ee/ExpressionEngine/Controller/Members/Profile/Access.php(192): ExpressionEngine\Controller\Members\Profile\Access->getPermissions()
> #1 ee/ExpressionEngine/Controller/Members/Profile/Access.php(36): ExpressionEngine\Controller\Members\Profile\Access->getAccessRow('can_view_online...')
> #2 [internal function]: ExpressionEngine\Controller\Members\Profile\Access->index()
> #3 ee/ExpressionEngine/Core/Core.php(255): call_user_func_array(Array, Array)
> #4 ee/ExpressionEngine/Core/Core.php(111): ExpressionEngine\Core\Core->runController(Array)
> #5 ee/ExpressionEngine/Boot/boot.php(161): ExpressionEngine\Core\Core->run(Object(ExpressionEngine\Core\Request))
> #6 public_html/edit.php(153): require_once('...')
> #6 public_html/edit.php(153): require_once('...')

<img width="1193" alt="Screen Shot 2021-10-19 at 1 08 05 PM" src="https://user-images.githubusercontent.com/20441129/137983193-5fbdba81-a838-4554-aef4-57fc16004829.png">

**Comment by ops-andy** [comment]
@mcfarlandonline Thanks for submitting this. I haven't been able to replicate this with just having a member with multiple roles and primary being locked. Any chance you can list out add-ons you have, if the member you were viewing only had certain access to add-ons, etc. Anything else that would stand out. thanks

**Comment by mcfarlandonline** [comment]
Hi Andy, thanks for checking into this. Now that I'm checking, I've also encountered this error with a member who's Primary Role is not locked, but one of the Additional Roles they're assigned to is the Site Admin/Locked group. 

Add-Ons we have installed for this site are:
DataGrab 3.0.3
Calendar 4.0.0
Query 2.0.0
Statistics 2.0.0
Streeng 1.10.0

For the locked "Site Admin" Role, we had "Access add-ons" turned on, but then none of the add-ons were checked under "Allowed add-on access". I tried turning off "Access add-ons" for that Role, but then got the same error/result when testing Access Overview with a member assigned to the Site Admin Role.
